### PR TITLE
[text-input] added space between sections

### LINKED ISSUE #5518: [Sistent] Add spacing between the sections
### Current Behavior
The spacing between the different sections on text-input page in Sistent is very less making it look cluttered.
image

### Desired Behavior
Add good amount of spacing between the sections.

### Implementation
1. Make sure to use rem for spacing
2. Spacing should be equal between each section

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
**Description**

This PR fixes #5518 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit eed19fdb7b3b20e115730a06317360e49fab1596 at: https://6670db64e1a0758140cdca2c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c921386017db132102d9a5171c41e6a073e19e4c at: https://6672625ba47af8ac0ed36b4b--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8aab64ba49cd113ffe58f8bb8f103b6c902447a8 at: https://6673b62edd68c8ef76fc23d5--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4da993041671f4c60a8f30c0c1c0c8aaeceace1b at: https://667887b0faf0e2f9900f7773--layer5.netlify.app

**Comment by Deepak0320Singhal** [comment]
@gagan-bhullar-tech Thank you for contribution.
Let's discuss this on Websites's call today 6:30 PM IST. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

**Comment by l5io** [comment]
🚀 Preview for commit c65cd06333578e6541086f05d2e8097c6971cc4f at: https://667b169f3ba8775d2f77403e--layer5.netlify.app

File: src/sections/Projects/Sistent/sistent.style.js [hunk 1]
```diff
@@ -611,6 +611,10 @@ const SistentWrapper = styled.div`
 
   .main-content {
     padding-top: 1rem;
+    a {
```
Comment (sudhanshutech) [inline src/sections/Projects/Sistent/sistent.style.js:614]: both div and anchor with padding why?

**Comment by iArchitSharma** [comment]
@gagan-bhullar-tech still working on this?

**Comment by gagan-bhullar-tech** [comment]
@iArchitSharma pushed one more change

**Comment by l5io** [comment]
🚀 Preview for commit cd1c123c3e1bdd595a5ba7ea59e49a90b71dac14 at: https://668b6817f27fe76e55de64c0--layer5.netlify.app

**Comment by sudhanshutech** [comment]
checking in last time...

**Comment by Ashparshp** [comment]
@gagan-bhullar-tech, thanks for the pr but as we haven't heard back from you.. if we don't hear back from you, unfortunately I'll be closing this issue...

**Comment by gagan-bhullar-tech** [comment]
@Ashparshp I have made the required changes long time back. I am not sure what else is needed in this PR. Please let me know what is the expectation here.

**Comment by Ashparshp** [comment]
@gagan-bhullar-tech, Fix DCO, sign-off please.

**Comment by Ashparshp** [comment]
@gagan-bhullar-tech, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by gagan-bhullar-tech** [comment]
@Ashparshp sorry, I don't have time to attend calls. I am closing PR.

**Pull request closed without merging**